azurerm/internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -171,6 +171,11 @@ func resourceKubernetesCluster() *schema.Resource {
 							Optional: true,
 							Computed: true,
 						},
+						"max_empty_bulk_delete": {
```

Comment:
could we call this `empty_bulk_delete_max`?

Reply:
@katbyte thanks! Updated as per your suggestion